config/runtime/fluster-chromeos.jinja2

```diff
@@ -0,0 +1,6 @@
+{%- extends 'chromeos.jinja2' %}
+
+{%- set test_method = 'fluster-chromeos' %}
```

Comment:
thank you @pawiecz . I reverted the last commit which was unifying templates and I also added the `params: test_method` field

Reply:
Sure, that works too. Now `test_method` is set twice: (1) in `pipeline.yaml` as `params: test_method=...` and (2) in this template here (which overrides (1) anyway).

Since this template is not used for any other set of Jobs let's leave it as it is now. It can be dealt with (along the template unification) as a follow-up PR once Fluster Jobs are merged.